staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -147,3 +147,21 @@ type TLSConfig struct {
 	// +optional
 	ClientCert string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// TracingConfiguration provides versioned configuration for tracing clients.
+type TracingConfiguration struct {
+	metav1.TypeMeta
+
+	// +optional
+	// URL of the collector that's running on the control-plane node.
+	// The APIServer uses the egressType ControlPlane when sending data to the collector.
+	// Defaults to localhost:4317
+	URL *string
```

Comment:
@dashpole the etcd config options seems to have a instance-id and a service-name:
https://github.com/etcd-io/etcd/pull/12919/files

would that be useful to us too?

Reply:
The service name should be a constant.  An instance id is a great idea.  It turns out there is already an APIServerID field we generate for other purposes, so i'll just use that.